### ISSUE TITLE
Reference journals/publishers that publish using GitHub

### ISSUE THREAD

**Issue by dhimmel** [greenelab/meta-review#14]
There are at least two:

```md
[ReScience](https://rescience.github.io/) [@arxiv:1707.04393v1]
[Open Journals](http://www.theoj.org/)
```

**Comment by agitter** [comment]
I added JOSS and ReScience to the Discussion in #6, but I wasn't aware of the other Open Journals.

**Comment by agitter** [comment]
Closed by #19
